deploy/vf-switcher.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
```

Comment:
Why need the service, vf switcher is a background job it is not needed at boot

Reply:
using services, it would be easier to start, stop, and collect logs across scripts. It would also be easier to debug in case of failures.